### PR TITLE
Delete Command (Tag Variant): Implement Parse and Execution

### PR DESCRIPTION
**Pull request by nbriannl**
Delete Command but works on Tags as well.

delete t/word1 t/word2 t/word3

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -18,42 +25,76 @@
...
 
 
     @Override
     public CommandResult executeUndoableCommand() throws CommandException {
-
-        List<ReadOnlyPerson> lastShownList = model.getFilteredPersonList();
-
-        if (targetIndex.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        if (targetTags.equals(null) && !targetIndex.equals(null)) {
```
Comment (April0616) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:55]: avoid to use 'null' in equals function, use == instead

**Comment by April0616** [comment]
need to update, so many tests failed...

**Comment by nbriannl** [comment]
@April0616 Yeah it's work in progress. Still ongoing

**Comment by nbriannl** [comment]
I will redo in another branch

**Pull request closed without merging**